packages/quantic/cypress/e2e/default-2/generatedAnswer/generated-answer.cypress.ts

```diff
@@ -136,6 +140,12 @@ describe('quantic-generated-answer', () => {
             Expect.generatedAnswerIsStreaming(false);
           });
 
+          it('should display the disclaimer', () => {
+            Expect.displayDisclaimer(true);
+            Expect.disclaimerContains(GENERATED_ANSWER_DISCLAIMER);
+            Expect.generatedAnswerIsStreaming(false);
+          });
+
```

Comment:
We should add the same test to the describe `describe('when the property multilineFooter is set to true', ...)` which can be found on line 215.